tgui/packages/common/react.ts

```diff
@@ -66,8 +63,14 @@ export const pureComponentHooks = {
 /**
  * A helper to determine whether the object is renderable by React.
  */
-export const canRender = value => {
+export const canRender = (value: unknown) => {
   return value !== undefined
     && value !== null
     && typeof value !== 'boolean';
 };
+
+/**
+ * A common case in tgui, when you pass a value conditionally, these are
+ * the types that can fall through the condition.
+ */
+export type BooleanLike = number | boolean | null | undefined;
```

Comment:
Why is this necessary? Why can't we just pass true/false and cut the truthiness?

Reply:
Byond sends 0/1 from the backend, so this is necessary to cut down on `Boolean()` or `!!` casts

Reply:
Ah, fair enough.

Might be a good case for enums, though?

Reply:
Could you povide an example?

Reply:
```ts
enum ByondBool {
    True = 1,
    False = 0,
}
```

Reply:
I see what you mean, but we should implement this later, I don't want to delay this PR further.

Reply:
👍